labkey-client-api/src/org/labkey/remoteapi/domain/QueryFilter.java

```diff
@@ -0,0 +1,65 @@
+package org.labkey.remoteapi.domain;
+
+import org.labkey.remoteapi.query.Filter;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class QueryFilter
+{
+    private List<Filter> _queryFilter;
+
+    public QueryFilter(List<Filter> filters)
+    {
+        _queryFilter = filters;
+    }
+
+    public List<Filter> getQueryFilter()
+    {
+        return Collections.unmodifiableList(_queryFilter);
+    }
+
+    public void setFilters(List<Filter> filters)
+    {
+        _queryFilter = filters;
+    }
+
+    static public QueryFilter fromJSONString(String filterStr)
+    {
+        List<Filter> queryFilter = new ArrayList<>();
+
+        String[] filterStrs = filterStr.split("&");
+        for (String filter : filterStrs)
+        {
+            String filterExp = filter.substring(filter.indexOf("~") + 1, filter.indexOf("="));
+            Filter.Operator op = Filter.Operator.getOperatorFromUrlKey(filterExp);
+
+            String value = filter.substring(filter.lastIndexOf("=") + 1);
+            queryFilter.add(new Filter((Object) value, op));
```

Comment:
Suggest replacing with:
- queryFilter.add(new Filter("format.column", value, op));

This way we can avoid the introduction of a new Filter constructor.